docs/proposals/20221109-managed-external-etcd.md

```diff
@@ -0,0 +1,572 @@
+---
+title: External etcd cluster lifecycle support
+authors:
+  - "@mrajashree"
+  - "@g-gaston"
+creation-date: 2022-11-09
+last-updated: 2022-11-09
+status: provisional
+---
+
+# External etcd cluster lifecycle support
+
+## Table of Contents
+
+- [Title](#title)
+  - [Table of Contents](#table-of-contents)
+  - [Glossary](#glossary)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+  - [Proposal](#proposal)
+    - [User Stories](#user-stories)
+    - [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+    - [Future work](#future-work)
+    - [POC implementation](#poc-implementation)
+    - [Questions](#questions)
+    - [Design questions](#design-questions)
+    - [Security Model](#security-model)
+    - [Risks and Mitigations](#risks-and-mitigations)
+  - [Alternatives](#alternatives)
+  - [Upgrade Strategy](#upgrade-strategy)
+  - [Additional Details](#additional-details)
+    - [Test Plan [optional]](#test-plan-optional)
+    - [Graduation Criteria [optional]](#graduation-criteria-optional)
+    - [Version Skew Strategy [optional]](#version-skew-strategy-optional)
+  - [Implementation History](#implementation-history)
+
+## Glossary
+
+### Etcd provider
+A provider that creates and manages an etcd cluster to be used by a single workload cluster for the [external etcd topology](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/ha-topology/#external-etcd-topology).
+
+### Etcdadm based etcd provider
+This provider is an implementation of the Etcd provider that uses [etcdadm](https://github.com/kubernetes-sigs/etcdadm) to create and manage an external etcd cluster.
```

Comment:
Last release was in 2021, is this still maintained?

Reply:
Yeah, it is. I myself got a couple PRs after that 2021 release. I suspect no one has asked for a release and that's why it hasn't been cut. But I can reach out to get one if we move ahead with this.